File: 2026-08/src/day_23/prms_181934/Solution.java
package day_23.prms_181934;

class Solution {
    public int solution(String ineq, String eq, int n, int m) {
        if (ineq.equals(">")) {
            if (eq.equals("=")) return n >= m ? 1 : 0;
            return n > m ? 1 : 0;
        } else {
            if (eq.equals("=")) return n <= m ? 1 : 0;
            return n < m ? 1 : 0;
        }
    }
}

File: 2026-08/src/day_23/prms_181934/Solution.cpp
#include <bits/stdc++.h>
using namespace std;

int solution(string ineq, string eq, int n, int m) {
    if (ineq == ">") {
        if (eq == "=") return n >= m;
        return n > m;
    } else {
        if (eq == "=") return n <= m;
        return n < m;
    }
}
